core/drivers/crypto/stm32/stm32_saes.c

```diff
@@ -155,13 +155,13 @@ static bool does_chaining_mode_need_iv(uint32_t cr)
 static bool is_encrypt(uint32_t cr)
 {
 	return (cr & _SAES_CR_MODE_MASK) ==
-	       SHIFT_U32(_SAES_CR_MODE_ENC, _SAES_CR_MODE_SHIFT);
+		SHIFT_U32(_SAES_CR_MODE_ENC, _SAES_CR_MODE_SHIFT);
```

Comment:
Nitpicking: can discard this change: revious indentation was fine (7 chars to match `return `).

Reply:
Ack